cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -261,10 +279,16 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   this.setCollapseTransition = setCollapseTransition;
   this.getTransition = getTransition;
   this.getDom = getDom;
+  this.isAnimating = isAnimating;
 
   // Public static properties.
   FlyoutMenu.EXPAND_TYPE = 'expand';
```

Comment:
Yeah the constant is `EXPAND_TYPE` and 'expand' is the value, so references to 'expand' are intended to use `EXPAND_TYPE`. This is a type-safe helper for `getTransition( FlyoutMenu.EXPAND_TYPE )`.